local_costmap:
  start_time: 5
  padding: 2 # number of grid cells to pad the c_scpace with
  local_width: 0.5 # width in meters of the local costmap
  local_height: 0.5 # height in meters of the local costmaps